gui/mozregui/pref_editor.py

```diff
@@ -30,13 +30,13 @@ def data(self, index, role=Qt.DisplayRole):
             if index.column() == 0:
                 return name
             else:
-                return repr(value)
+                return QString(value).__str__()
 
     def flags(self, index):
         return Qt.ItemIsSelectable | Qt.ItemIsEditable | Qt.ItemIsEnabled
 
     def setData(self, index, new_value, role=Qt.EditRole):
-        new_value = str(new_value.toString())
+        new_value = new_value.toPyObject().__str__()
```

Comment:
I'm totally fine with unicode(new_value.toString()) - I find it better for readability too.